### ISSUE TITLE
Timeslider labels - Create option to hide or disable them

### ISSUE THREAD

**Issue by dan-bowerman** [ramp4-pcar4/storylines-editor#300]
## Description

![image](https://github.com/ramp4-pcar4/storylines-editor/assets/5294596/423ecfdd-8867-4801-a83e-d41c3d9587ee)

If a timeslider has too many notches on it, labels can overlap and look messy. Since the value ranges are already described by the label above the slider, perhaps we can have an option to omit labels entirely, showing only the notches. 

## Alternatives considered

Open to ideas!

**Comment by james-rae** [comment]
Less visually impressive but I imagine would satisfy a11y. The combos would render upwards when opened on the site (i'm just being lazy with mockups)

![image](https://github.com/ramp4-pcar4/storylines-editor/assets/7585245/1a4c7299-dca3-4a3c-accf-0dfb6fcc9550)

**Comment by dan-bowerman** [comment]
From James in the Slack chat:

> https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child
> So if there's a class targeting whatever contains all the slider ticks, you could probably do something like (CSS will be wrong, just the gist of it)

```
.listTickClass {
  display: none;
}

.listTickClass:first-child, .listTickClass:last-child {
  display: allgood;
}
```

Spencer replies:
> What James suggested (only first and last label) was done for https://climate-viewer.canada.ca/#/?v=rdpa6&d=capa&cp=-95.80098000546008,59.67409464609375&z=2&ts=1715904000000 and I think it works well.
> If we want more labels I remember it being fairly straightforward to have the slider library show every Nth label. Could also be configurable.

**Comment by dan-bowerman** [comment]
An easy Editor-friendly workaround is to stick the following snippet into any slide:
`<style>.time-slider .noUi-value-horizontal { display: none; }</style>`

Be advised: This will remove all timesliders labels on the page. There don't appear to be any WCAG or functional consequences to hiding these labels.
